.drone.yml

```diff
@@ -0,0 +1,17 @@
+kind: pipeline
+type: docker
+name: default
+
+steps:
+  - name: build
+    image: node:12.7.0-alpine
+    commands:
+      - apk add --no-cache zip curl
+      - yarn
```

Comment:
Can we add Dockerfile so that Node dependencies and cli tools can be cached in order to have a shorter build time?